spec/lib/middleware/websocket/interactors/client_creation_payload_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe Websocket::Interactor::ClientCreationPayload do
+  let(:client_1) do
+    Websocket::Client.new(connection: double('connection').as_null_object)
+  end
+  let(:client_2) do
+    Websocket::Client.new(connection: double('connection').as_null_object)
+  end
+  let(:clients) { [client_1, client_2] }
+
+  describe '#call' do
+    subject { described_class.new.call(client: client_1) }
+
+    before do
+      allow(client_1.player).to receive(:id)
+      allow(client_1.player).to receive(:token)
```

Comment:
Is this stubbing necessary?

Reply:
@jgayfer same with below comment, [is this better?](https://github.com/daelynj/typinggame-server/pull/28/commits/acab9e3831b728727b7aa1d43ff0086b407c7215)